config/search/config.json

```diff
@@ -363,6 +363,155 @@
           }
         }
       }
+    },
+    {
+      "type": "newsletter-infos",
+      "on_path": "newsletter-infos",
+      "rdf_type": "http://data.vlaanderen.be/ns/besluitvorming#NieuwsbriefInfo",
+      "properties": {
+        "richtext": "http://mu.semte.ch/vocabularies/ext/htmlInhoud",
+        "subtitle": "http://dbpedia.org/ontology/subtitle",
+        "title": "http://purl.org/dc/terms/title",
+        "publicationDate": [
+          "^http://www.w3.org/ns/prov#generated",
+          "http://data.vlaanderen.be/ns/besluitvorming#heeftOnderwerp",
+          "^http://purl.org/dc/terms/hasPart",
+          "http://data.vlaanderen.be/ns/besluitvorming#isAgendaVoor",
+          "http://data.vlaanderen.be/ns/besluit#geplandeStart"],
+        "decisionStatusId": [
```

Comment:
I propose to rename `decisionStatusId` to `decisionResultCodeId`, in line with how this concept is named in [other parts of the codebase](https://github.com/kanselarij-vlaanderen/kaleidos-project/blob/4263033bfb2ddcfb46ce66e08fe5b4051ecc01dd/config/resources/besluit-domain.lisp#L126)

Reply:
renamed